apps/web/src/pages/quick-start/steps/Quickstart.tsx

```diff
@@ -39,6 +38,8 @@ export function Quickstart() {
   );
 }
 
+const BellGradientIcon = () => <BellGradient style={{ width: '40px', height: '40px' }} />;
```

Comment:
our icons are looking wrong when you do inspect its viewBox... that's why I had to do this hack in order to have them look the same... I'll talk to Nikolay about it